src/devices/RelayBoard/README.md

```diff
@@ -0,0 +1,36 @@
+﻿# RelayBoard
+
+## Summary
+This is a basic class for controlling Relays. It also provides a convenient ``RelayBoard`` class that allows you to easily manage the different relay boards on your system.
+
+## Device Family
+This library should work with any basic relay that takes one input pin, including both normally closed and normally open relays.
+
+## Binding Notes
+
+You can either use the ``RelayBoard`` class to help organize multiple relays (especially if they are of the same type), or use the individual ``Relay`` class directly.
+
+If you use ``Relay`` directly, you *must* manage the lifetime of the GPIO controller.
+
+An example of how you would use a ``RelayBoard``:
+
+```csharp
+// Create a relay board, using default values
+var board = new RelayBoard();
```

Comment:
in a sample you should dispose the relay as well to avoid teaching anti-patterns